packages/e2e/cypress/integration/ui/components/authenticator/reset-password/reset-password.steps.ts

```diff
@@ -1,5 +1,28 @@
-import { Then } from 'cypress-cucumber-preprocessor/steps';
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
 
 Then('I will be redirected to the confirm forgot password page', () => {
-  cy.findByRole('textbox', { label: 'New password' }).should('exist');
+  cy.findInputField('New Password').should('exist');
+});
+
+When('I type my new password', () => {
+  cy.findInputField('New Password').type(Cypress.env('VALID_PASSWORD'));
+});
+
+Then('I click the submit button', () => {
+  /**
+   * Submit button text differs on React/Vue vs Angular. Testing for both for
+   * now, but we can look to make them consistent on next major release.
+   */
+  cy.findByRole('button', {
+    name: new RegExp(`^((submit)|(send code))$`, 'i'),
```

Comment:
Again, inconsistent labels across framework. Leaving this as now because changing labels is technically a breaking change for those who use I18n. We can make them consistent in next breaking release.

Reply:
Can you create a ticket for this and mark it as a breaking chance?